server/app.ts

```diff
@@ -22,6 +22,11 @@ export default function createApp(): express.Application {
   app.set('trust proxy', true)
   app.set('port', process.env.PORT || 3000)
 
+
+  app.use(express.urlencoded({ extended: false }))
+
+  app.use(appInsightsMiddleware())
```

Comment:
The app insights functionality has been removed so this call to middleware likely won't work